balancer/wrr/wrr.go

```diff
@@ -0,0 +1,9 @@
+package wrr
+
+// Name is the name of weighted_round_robin balancer.
+const Name = "weighted_round_robin"
+
+// Information that should be stored inside Address metadata in order to use wrr.
+type Info struct{
```

Comment:
Maybe let's make it an interface, e.g.
```
type Weighted interface {
  Weight() uint32
}
```
?

This would allow us to provide some object with xds-specific knowledge (e.g. is it a canary instance?) in the future, which would also implement the `Weighted` interface among others.

Reply:
Also it may make sense to move this interface to the `balancer` package itself, as it's possible it would be used by other weight-aware load balancers, e.g. maglev.

Reply:
The problem with interface is, it's hard to add methods. But for structs, we can add fields.

I also think it's better to not keep it in `balancer` package, because it's more like a feature of a specific balancer, not all balancers.